src/traces/sankey/render.js

```diff
@@ -444,11 +444,11 @@ module.exports = function(svg, styledData, layout, callbacks) {
         .append('g')
         .classed('sankeyLinks', true)
         .style('fill', 'none')
-        .style('transform', linksTransform);
```

Comment:
Good catch. 

Would anyone be interested in making @alexcjohnson 's [`strict-d3`](https://github.com/plotly/plotly.js/blob/master/test/image/strict-d3.js) wrapper throw an error on `selection.style('transform', <>)`?